src/client/app/app-routing.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+
+@NgModule({
+  imports: [
+    RouterModule.forRoot([
+      /* define app non-feature routes here */
```

Comment:
More of a general question: Since we both have the `HomeModule` and `AboutModule`, which routes would be considered 'non-feature' routes? Sure, at the moment we don't have any. But in a broader context, what routes would be defined in the `AppRoutingModule` ?